data-esnext.js

```diff
@@ -91,6 +91,14 @@ exports.tests = [
       name: 'basic support',
       mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SIMD',
       exec: function () {/*
+        simdFloatTypes=['Float32x4'];
```

Comment:
I was under the impression that if I declared them without the preceding "var" they would have global scope.  And they seem to be working like this: all the tests still pass, but they fail if I use the wrong array on a given test.